whylog/tests/tests_front/tests_whylog_factory.py

```diff
@@ -0,0 +1,13 @@
+from whylog.front.whylog_factory import whylog_factory
+from whylog.log_reader import LogReader
+from whylog.teacher import Teacher
+from whylog.tests.utils import TestRemovingSettings
+
+
+class TestWhylogFactory(TestRemovingSettings):
+    def tests_whylog_factory(self):
+        log_reader, teacher_generator = whylog_factory()
+        teacher = teacher_generator()
+
+        assert isinstance(log_reader, LogReader)
```

Comment:
This test tests only whylog_factory which creates Only LogReader. We don't have implemented ThreadedLogReader yet and we don't have implemented LogReaderSelector or something like this. I only use the interface of LogReader that is available. If something is wrong with it give comments not to me but to @konefalg .